robot/stream/octet.robot

```diff
@@ -44,21 +47,17 @@ Start Key Value Store Server
     Wait For Server Startup
 
 Start Text Client
-    [Arguments]   ${stdin}
     ${hdl}=  Start Process
     ...  ${CLIENT_PATH}
     ...  -p    ${SERVER_PORT}
     ...  --caf.logger.file.path  text-store.log
     ...  stdin=PIPE
     ...  stdout=text-store.out
     ...  stderr=text-store.err
-    # Note: we can't pass ${stdin} directly as stdin=${stdin} because this would
-    # close stdin immediately after the process has read the input. This would
-    # in turn cause the process to exit immediately without waiting for the
-    # server to respond.
-    Evaluate  $hdl.stdin.write($stdin.encode())
-    Evaluate  $hdl.stdin.flush()
-    [Return]  ${hdl}
+
+Send Message
+    [Arguments]    ${process}   ${message}
+    Write To Process Stdin    ${process}    ${message}
```

Comment:
I have put these in separate python files as they are used by multiple robot tests. I think this can be used by other robot tests in future too. The alternative way is fine too, just will result in some duplicated statement.

Reply:
Please remove that Python file again. It's just two lines, so I definitely prefer having those two lines in the code as opposed to spreading the logic across multiple files.